@navikt/ds-react/src/header/Header.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { forwardRef, HTMLAttributes } from "react";
+import cl from "classnames";
+
+export interface HeaderProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Component content
+   */
+  children: React.ReactNode;
```

Comment:
Skulle vi ha begrenset tittelen til en string for å unngå at man setter inn andre komponenter, eller tenker du at det er lurt å holde det mer åpent ved å godkjenne ReactNode? Dere kjenner jo godt til hvordan team bruker felleskomponentene etc så jeg bare lurer jeg, altså.

Reply:
Jeg er helt med på å begrense den til `string`. Jeg ser for meg at denne komponentene skal være relativt "standard", og trenger da ikke så mye fleksibilitet.